components/molecules/Header.js

```diff
@@ -0,0 +1,130 @@
+import propTypes from 'prop-types'
+import Link from 'next/link'
+
+import { useRouter } from 'next/router'
+
+import en from '../../locales/en'
+import fr from '../../locales/fr'
+
+export default function Header({ language }) {
+  // const language = items.language;
+  const t = language === 'en' ? en : fr
+
+  const router = useRouter()
+
+  return (
+    <>
+      <nav
+        role="navigation"
+        aria-label={t.skipToMainContent}
+        className="skip-main"
+      >
+        <a
+          id="skipToMainContent"
+          className="bg-custom-blue-dark text-white py-1 px-2 focus:outline-black-solid hover:bg-gray-dark"
+          href="#wb-cont"
+          draggable="false"
+        >
+          {t.skipToMainContent}
+        </a>
+      </nav>
+
+      <header>
+        <div className="layout-container flex-col flex md:flex md:flex-row justify-between mt-2">
+          <div className="flex flex-row justify-between items-center lg:mt-7 mt-1.5">
+            <a href={t.gocLink}>
+              <img
+                className="h-5 w-auto xs:h-6 sm:h-8 md:h-8 lg:h-7 xl:h-8"
+                src={language === 'en' ? '/sig-blk-en.svg' : '/sig-blk-fr.svg'}
+                alt={
+                  language === 'en'
+                    ? 'Government of Canada'
+                    : 'Gouvernement du Canada'
+                }
+              />
+            </a>
+
+            {/* Language selector for small screens */}
+            <Link
+              key={language}
+              href={router.asPath}
+              locale={language === 'en' ? 'fr' : 'en'}
+            >
+              <a
+                className="visible md:invisible md:text-sm ml-6 sm:ml-16 underline font-body font-bold text-canada-footer-font  text-base hover:text-canada-footer-hover-font-blue"
```

Comment:
It's better to use display hidden instead of invisible.
https://tailwindcss.com/docs/display

This makes the element not take up space on the page

Reply:
https://tailwindcss.com/docs/visibility they do recommend

Reply:
They recommend both but invisible takes up space
![image](https://user-images.githubusercontent.com/69208837/136077237-b20c03ea-8406-48e6-8958-19274b58f9f1.png)